### ISSUE TITLE
Add an Item – Tags – Duplicate Tag Entry Lacks Warning Status

### ISSUE THREAD

**Issue by aviannamiller** [omeka/Omeka#1131]
"Description: On the Add Items page under the 'Tags' tab, it was found that adding duplicate tags lacks a proper warning. Currently, when adding tags, there is no status message, noted in defect 136. But when users attempt to add the same tag as what is listed below the ‘All tags’ heading, there is no warning that users cannot add duplicate tags, and this should be represented in text form as well as announced to the users. This isn’t a true ‘error’ due to the tags not being required but does impact users who might not realize they are adding the same tag in this flow. 

Expected Results: When users attempt to add a duplicate tag there is a status message and text populated that warns the uses that duplicate tags are not added. 

Actual Results: When users attempt to add a duplicate tag the edit field is cleared, there isn’t a status message or text populated on the page informing users that the tag was not added"

3.3.1 Error Identification; Level A; WCAG 2.0
